test/CMakeLists.txt

```diff
@@ -716,7 +716,7 @@ if(WORKAROUND_ISSUE_936 AND MIOPEN_TEST_HALF)
     LIST(APPEND IMPLICITGEMM_ARGS --disable-forward --disable-backward-data)
     #Afther fix need to remove '| grep -v "cannot be executed due to incorrect params"'
 endif()
-add_custom_test(test_conv_for_implicit_gemm SKIP_UNLESS_ALL BF16_ENABLED HALF_ENABLED GFX1030_ENABLED
+add_custom_test(test_conv_for_implicit_gemm SKIP_UNLESS_ALL BF16_ENABLED HALF_ENABLED GFX90A_ENABLED GFX1030_ENABLED
```

Comment:
AFAIU this won't pass anyway, because non-xdlops solvers are not enabled for gfx90a. That is why this test is not enabled for MI100.